app/src/componentLibrary/systemMessage/systemMessage.scss

```diff
@@ -0,0 +1,58 @@
+@mixin font-config($weight, $size, $height, $color) {
+  font-weight: $weight;
+  font-size: $size;
+  line-height: $height;
+  color: $color;
+}
+
+@mixin message-headers($color) {
+  color: $color;
+  text-transform: capitalize;
+  margin-bottom: 8px;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.system-message-container {
+  min-height: 104px;
+  height: auto;
+  width: 100%;
+  background: $COLOR--bg-000;
+  border-radius: 8px;
+  font-family: $FONT-ROBOTO-REGULAR;
+  box-shadow: 0px 1px 3px rgba(55, 67, 98, 0.1);
```

Comment:
Please extract box-shadow to corresponding .scss file in common variables with name from design